trellis-audit/src/main/java/org/trellisldp/audit/DefaultAuditService.java

```diff
@@ -42,13 +38,9 @@
  * objects in a {@code http://www.trellisldp.org/ns/trellis#PreferAudit} named
  * graph.
  * 
- * @implSpec This implementation of AuditService does not persist audit
- *           information. Subclass and override {@link #add(Resource)} to add
- *           persistence.
- * 
  * @author acoburn
  */
-public class DefaultAuditService implements AuditService {
+public abstract class DefaultAuditService implements AuditService {
```

Comment:
Why is this abstract? Hasn't the `::add` method been moved elsewhere?

Reply:
Yup, looks like an oversight which I will now fix.